### DISCUSSION TITLE
Not able to install on PHP version 8.2.*

### DISCUSSION THREAD

**Discussion by askaralimkndr** [spatie/laravel-backup#1682] [General]
"Hello, I couldn't find any other packages for database backup that specifically support PHP version 8.2. Hopefully, more packages will be upgraded to support this version soon. Best regards."

**Comment by angeljqv** [comment]
https://github.com/spatie/laravel-backup/blob/255b17479abda0284f57dae3b98b7cfac38afdea/composer.json#L21
`"^8.0"` includes 8.2

**Reply by askaralimkndr** [reply]
Still not solved the installation issue, can you please provide me more details? how can I proceed to install laravel-permission package in my laravel 10 application which is running on PHP 8.2 ?

Thank you

**Reply by angeljqv** [reply]
It seems you don't understand how things work, I don't have any problem, how could I give you details?
You are supposed to give the details, you don't even post the error you get, what version of the package are you trying to install, what operating system
And yet you hope that someone has an idea that you are doing wrong when it works for everyone
you will have to wait for a collaborator who has psychic powers 😄😄
